Lib/fontTools/varLib/__init__.py

```diff
@@ -1019,12 +1092,12 @@ def main(args=None):
 	logging_group = parser.add_mutually_exclusive_group(required=False)
 	logging_group.add_argument(
 		"-v", "--verbose",
-                action="store_true",
-                help="Run more verbosely.")
+				action="store_true",
+				help="Run more verbosely.")
```

Comment:
Why?

Reply:
Mixture of tabs and spaces. This tabs everything.